### PR TITLE
Add information about various possible search targets

### PR DESCRIPTION
**Pull request by abhi18av**
Tweak the search information to mention possible search targets.

### CODE REVIEW

File: docs/monitoring/overview.md [hunk 1]
```diff
@@ -25,9 +25,7 @@ In the left bar:
 
 ## Search
 
-The search box enables searching for runs and pipelines by name. 
-
-Use asterisks `✽` before and after keyword to filter results.
+The search box allows searching for workflows by `project name`, `run name`, `session id` or `manifest name`. Moreover, wildcards can be used to filter the desired workflows such as using asterisks `✽` before and after keyword to filter results.
```
Comment (pditommaso) [inline docs/monitoring/overview.md:28]: From where come from this snowflake char `✽`!! 😆

Comment (abhi18av) [reply docs/monitoring/overview.md]: Hawk-eyes again! 😄 

It was already there, we can call it a `legacy snowflake`. Anyways, I've updated this now.

**Comment by pditommaso** [comment]
Keeping on-hold until the the search improvements are released

**Comment by abhi18av** [comment]
@pditommaso , is this PR all set to be merged?

**Comment by pditommaso** [comment]
Fair enought

**Pull request merged**